services/github/src/handlers/job.js

```diff
@@ -28,7 +28,12 @@ async function jobHandler(message) {
   let repository = message.payload.repository;
   let sha = message.payload.details['event.head.sha'];
   debug = debug.refine({ owner: organization, repo: repository, sha });
-  let pullNumber = message.payload.details['event.pullNumber'];
+  let pullNumber = message.payload.details['event.pullNumber'] || message.payload.body.number;
+
+  debug(`Retrieving  ${organization}/${repository}@${sha}...`);
```

Comment:
Do you still want this debug?

```suggestion
  debug(`Retrieving ${organization}/${repository}@${sha}...`);
```

Reply:
Ah I see it was a copy/paste

Reply:
eagle-eye :) thanks!